### ISSUE TITLE
v0.3.0-beta.4  - the code does not update on Source-->open in main area  section

### ISSUE THREAD

**Issue by yimmas** [jupyterlab/debugger#503]
With v0.3.0-beta.4 when i  run a code & set a breakpoint it stops where i intended. I can see the highlighted code with the breakpoint on the lower right  section under  "SOURCE".  

After that i can move to the next line of  the code via call stack section by pressing Next button. When i do that  the source section at the lower right updates with highlighting next line of the code. This is an expected behaviour.

**but when i click  Source-->open in main area**, the code opens up in the main area, **it stops  highlighting & moving to the next  line**. ( the source section on the lower right corner updates/highlighting/moving to the next line).

Basically the **Source-->open in main area does not refresh properly**. This does not happen on  version 0.2.0

I think that this is a bug and needs to be fixed.

Thanks

**Comment by jtpio** [comment]
Thanks @yimmas.

It is indeed a regression with the latest v0.3.0-beta.4, which should be fixed by https://github.com/jupyterlab/debugger/pull/501 and available in the next version.

**Comment by yimmas** [comment]
Thank you very much for a quick response.
